server/GameController.js

```diff
@@ -29,6 +28,55 @@ module.exports = function(data) {
 	//Number of white cards a user should always have
 	var HANDSIZE = 7;
 
+    /*
+    GameState:
+        0 : game hasn't started yet
+        _1 : game started, round _, everyone has to submit a choice
+        _2 : round _, everyone has to submit a vote
+        _3 : round _, votes are in
+        4: final results, game over
+    */
+    var GameState = '0';
+    var GameStateHasChanged = false;
+	var count = 30;
+
+	var callBackReference = null;
+
+    var startTimer = function(callback) {
```

Comment:
a lot actually got changed as i merged ben's version into mine. It's not finished yet, I'll add comments once i get everything working and i push the final version